codes/scala/chapter_computational_complexity/iteration.scala

```diff
@@ -0,0 +1,81 @@
+
+/**
+ * File: iteration.scala
+ * Author: WangBlue (wangjiping596@gmail.com)
+ * Create: 2024-08-09 10:50
+ */
+
+
+class iteration {
```

Comment:
Can we remove the `class` of these functions? (The same goes for the code in the other files).

We do this in Java because all functions should belong to a class, which is uncommon in other programming languages.